__tests__/sinopiaApi.test.js

```diff
@@ -58,6 +58,17 @@ afterEach(() => {
   global.fetch = originalFetch
 })
 
+describe('getGroups', () => {
+  it('retrieves list of groups', async () => {
+    const result = await getGroups()
+    expect(result.data).toEqual(expect.arrayContaining([
+      { id: 'stanford', label: 'Stanford University' },
+      { id: 'cornell', label: 'Cornell University' },
+    ]))
+    expect(result.data.length).toBeGreaterThanOrEqual(25)
+  })
+})
+
```

Comment:
this test fails, showing `result` as undefined, indicating that the `getGroups` function in `sinopiaApi` isn't working, and I don't know why ....

this then causes test failures throughout the stack